charts/kube-prometheus-stack/hack/sync_prometheus_rules.py

```diff
@@ -60,7 +61,7 @@ def new_representer(dumper, data):
         'min_kubernetes': '1.14.0-0'
     },
     {
-        'source': 'https://raw.githubusercontent.com/etcd-io/website/master/content/en/docs/v3.4/op-guide/etcd3_alert.rules.yml',
+        'source': 'https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/main/manifests/etcd-prometheusRule.yaml',
```

Comment:
cannot find https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/main/manifests/etcd-prometheusRule.yaml
404: Not Found
is the file still provided by kube-prometheus?

Reply:
ETCD monitoring is not enabled by default in kube-prometheus and thus rules are not generated by default and not included in `manifests/`.

Reply:
As a draft, I did not intend for this to be reviewed at this time.

As of right now, the current alerting rules for etcd are provided as a [jsonnet mixin](https://github.com/etcd-io/etcd/tree/main/contrib/mixin). At some point, the mixin needs to be instantiated somewhere, so I thought [`kube-prometheus`](https://github.com/prometheus-operator/kube-prometheus/pull/1769) seemed appropriate. However, @paulfantom does not think that the etcd alerting rules should be rendered at all by default, and thus they will not be included in the `manifests/` directory.

How can we get a set of current etcd alerting rules in to this project? The rules currently being pulled in are broken and obsolete.

Reply:
We had a similar conversation in https://github.com/etcd-io/etcd/pull/13671#pullrequestreview-875165715 
@paulfantom what do you think about the idea of kube-prometheus providing release artifacts that contain all the rendered manifests? So we could fetch a tarball here and do the necessary post-processing (assuming kube-prometheus be will released regularly).

Reply:
> @paulfantom what do you think about the idea of kube-prometheus providing release artifacts that contain all the rendered manifests?

The idea SGTM.

Reply:
Is this something I can do, or does a project maintainer have to do it?

Reply:
if possible, can we advance on it ? it's anoying to enable etcd and have broken alert templates to handle :)